docs/quickstart.rst

```diff
@@ -2,54 +2,17 @@
 Quick start
 =============
 
-Example locustfile.py
-=====================
+In Locust you define your user behaviour in plain Python code. You then use the ``locust`` command and (optionally) its web interface to spawn and simulate a number of those users while gathering request statistics.
 
-When using Locust you define the behaviour of users in Python code, and then you have the ability to 
-simulate any number of those users while gathering request statistic. The entrypoint for defining the 
-user behaviour is the `locustfile.py`.
-
-.. note::
-
-    The ``locustfile.py`` is a normal Python file that will get imported by Locust. Within it you 
-    can import modules just as you would in any python code.
-    
-    The file can be named something else and specified with the `-f` flag to the ``locust`` command.
-
-Below is a quick little example of a simple **locustfile.py**:
-
-.. code-block:: python
-    
-    import random
-    from locust import HttpUser, task, between
-    
-    class WebsiteUser(HttpUser):
-        wait_time = between(5, 9)
-        
-        @task(2)
-        def index(self):
-            self.client.get("/")
-            self.client.get("/ajax-notifications/")
-        
-        @task(1)
-        def view_post(self):
-            post_id = random.randint(1, 10000)
-            self.client.get("/post?id=%i" % post_id, name="/post?id=[post-id]")
-        
-        def on_start(self):
-            """ on_start is called when a User starts before any task is scheduled """
-            self.login()
+locustfile.py
+=============
+.. literalinclude:: ../examples/locustfile.py
         
-        def login(self):
-            self.client.post("/login", {"username":"ellen_key", "password":"education"})
-
-
-Let's break it down:
---------------------
+.. rubric:: Let's break it down
 
 .. code-block:: python
 
-    class WebsiteUser(HttpUser):
+    class MyUser(HttpUser):
```

Comment:
I don't see why `MyUser` would be a better name than `WebsiteUser`? I think it's better to use examples that more closely resembles some real world scenario, than to use very generic names.

Reply:
A good name would be FooUser if the site were named Foo, but we dont know what the name of the site is... Maybe QuickstartUser?

WebsiteUser(HttpUser) is just cookie-on-cookie ;) I just changed it to make it clear it is not important.

Reply:
Ah, I was thinking of `WebsiteUser` as opposed to `ApiUser`, but I see what you mean :). `QuickstartUser` sound ok, or maybe `DesktopUser` or `MobileUser`?

Reply:
I'll go with QuickstartUser if that is ok.